src/test/java/io/zold/api/RtTransactionTest.java

```diff
@@ -51,9 +62,31 @@ public void idIsNotYetImplemented() {
         new RtTransaction("id()").id();
     }
 
-    @Test(expected = UnsupportedOperationException.class)
-    public void timeIsNotYetImplemented() {
-        new RtTransaction("time()").time();
+    @Test
+    public void returnsTime() throws IOException {
```

Comment:
@carlosmiranda would be good to include in the name of the test that we are expecting iso 8601 format, so that it is clear it is a requirement and not a design choice (actually, maybe this is only what was needed and not making it explicit it in the implementation, sorry about that).
`returnsTimeInIso8601Format` would be perfect.